addons/im_livechat/models/discuss_channel.py

```diff
@@ -168,12 +168,21 @@ def _get_channel_history(self):
         """
         Converting message body back to plaintext for correct data formatting in HTML field.
         """
-        return Markup("").join(
-            Markup("%s: %s<br/>")
-            % (message.author_id.name or self.anonymous_name, html2plaintext(message.body))
-            # sudo: discuss.channel: can read all previous messages when converting to lead
-            for message in self.sudo().message_ids.sorted("id")
-        )
+        self.ensure_one()
+        parts = []
+        # sudo: res.partner: accessing chat bot partner is acceptable to build channel history.
+        chatbot_op = self.sudo().chatbot_message_ids[
+            :1
+        ].script_step_id.chatbot_script_id.operator_partner_id
+        last_msg_from_chatbot = False
+        for message in (self.message_ids - self.message_ids._filter_empty()).sorted("id"):
+            if message.author_id == chatbot_op and not last_msg_from_chatbot:
+                parts.append(Markup("<br/>"))
+            msg_template = "<strong>%s</strong>" if message.author_id == chatbot_op else "%s"
+            parts.append(Markup(f"{msg_template}<br/>") % html2plaintext(message.body))
```

Comment:
For good practice, markup should be around strong and br, not whole msg_template